ecosystem/typescript/sdk/src/aptos_types/account_address.ts

```diff
@@ -47,6 +47,31 @@ export class AccountAddress {
     return new AccountAddress(res);
   }
 
+
+  /**
+   * Checks if the string is a valid AccountAddress
+   * @param addr Hex string can be with a prefix or without a prefix,
+   *   e.g. '0x1aa' or '1aa'. Hex string will be left padded with 0s if too short.
+   */
+  static isValid(addr: MaybeHexString): boolean {
```

Comment:
might be good to add 
- check for 64 (or 66) chars long
- the string format, something like `/^(0[xX])?[a-fA-F0-9]{1,64}$/`

Reply:
It can be less than 64 characters, so that's not really to check right now

Reply:
a valid account address is less than 64 chars?

Also, not really a fan of padding the address with leading 0s if its too short, that can potentially make an invalid address into a valid address which it shouldn't. for example, If someone types/copy-past it wrong and missing some chars, thats not a valid address and we should be blocking it.